File: Frontend/src/infrastructure/dtos/vvn/unloadVVNCreateDto.ts
import type { UnloadCargoManifestDto } from "../cargo-manifest/unloadCargoManifestDto";
import type { VVNCreateDto } from "./vvnCreateDto";

export type UnloadVVNCreateDto = VVNCreateDto & {
  cargoManifestDTO: UnloadCargoManifestDto;
};

File: JWP_API/JadeWesserPort/DTOs/VVNDTOs/UnloadVVNCreateDTO.cs
using JadeWesserPort.DTOs.CargoManifestDTOs;

namespace JadeWesserPort.DTOs.VVNDTOs;

public class UnloadVVNCreateDTO : VVNCreateDTO
{
    public UnloadCargoManifestDTO CargoManifestDTO { get; set; } = null!;
}
